integrations/langchain4j/providers/cohere/pom.xml

```diff
@@ -127,6 +147,11 @@
                             <artifactId>helidon-common-features-processor</artifactId>
```

Comment:
You have changed to new Features API, but you did not change the annotation processor path. It must be `helidon-common-features-codegen`